### PR TITLE
Add argmin to namespace functions in Jax frontend

### PR DESCRIPTION
**Pull request by kojomensahonums**

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py [hunk 1]
```diff
@@ -1132,3 +1132,35 @@ def test_jax_numpy_arcsinh(
...
+    
+    
+# argmin
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float")),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.jax.numpy.argmin"
+    ),
+    keepdims=st.booleans(),
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:1144]: You are missing the axis argument. You can use `dtype_values_axis` from `helpers` instead of `dtype_and_values`.

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py [hunk 2]
```diff
@@ -1132,3 +1132,35 @@ def test_jax_numpy_arcsinh(
...
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="jax",
+        fn_tree="numpy.argmin",
+        a=np.asarray(x, dtype=input_dtype),
+    )
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:1165]: And then you should also pass `axis` and `keepdims` arguments here so that they are used by the frontend function 🙂

File: ivy/functional/frontends/jax/numpy/name_space_functions.py [hunk 3]
```diff
@@ -224,6 +224,10 @@ def fmax(x1, x2):
     return ret
```
Comment (AnnaTz) [inline ivy/functional/frontends/jax/numpy/name_space_functions.py:226]: Could you add the `@inputs_to_ivy_arrays` decorator here please?

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/jax/numpy/name_space_functions.py (hunk 3 above)
Comment (kojomensahonums) [reply ivy/functional/frontends/jax/numpy/name_space_functions.py:226]: I added the other arguments into the return ivy function. Hope that works?

**Review by AnnaTz — APPROVED** [review]

**Comment by AnnaTz** [comment]
Well done, ready to be merged 🙂

**Comment by kojomensahonums** [comment]
fixes #5403

**Comment by kojomensahonums** [comment]
argmin #5403

**Comment by kojomensahonums** [comment]
Closes #5403
